.prettierrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
Is this needed?

Reply:
My default formatter formats YAML with double quotes, so just wanted to ensure that standards.

Reply:
Let me know if you want me to remove it